style.css

```
@@ -0,0 +1,157 @@
+html {
+    font-family: 'Open Sans', sans-serif;
+    height: 100%;
+}
+
+/* MAIN CONTENT */
+
+body {
+    display:flex;
```

Consistent format throughout our code is great for ensuring clean code. Check out linters such as Prettier to automatically format your code for you on save. 
```suggestion
    display: flex;
```